src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportService.java

```diff
@@ -209,31 +234,35 @@ public void auditSomuExport(LocalDate from, LocalDate to, OutputStream output, S
         }
         headers.addAll(somuHeaders);
 
-        try (CSVPrinter printer = new CSVPrinter(outputWriter, CSVFormat.DEFAULT.withHeader(headers.toArray(new String[headers.size()])))) {
-            Stream<AuditData> data = auditRepository.findAuditDataByDateRangeAndEvents(
-                    LocalDateTime.of(from, LocalTime.MIN),
-                    LocalDateTime.of(to, LocalTime.MAX),
-                    SOMU_TYPE_EVENTS,
-                    caseTypeCode);
-
+        try (CSVPrinter printer = new CSVPrinter(outputWriter, CSVFormat.DEFAULT.withHeader(headers.toArray(new String[0])))) {
+            Stream<AuditData> data = getAuditDataStream(false, SOMU_TYPE_EVENTS, from, to, caseTypeCode);
             ExportDataConverter exportDataConverter = convert ? exportDataConverterFactory.getInstance() : null;
-
             data.forEach((audit) -> {
+                try {
+                    Thread.sleep(1000);
```

Comment:
Needed?

Reply:
Nope, left over from testing, glad you caught that